File: launchdarkly/typescript-node/src/UpdateFlagLinkExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.FlagLinksBetaApi();
apiCaller.setApiKey(api.FlagLinksBetaApiApiKeys.ApiKey, "YOUR_API_KEY");

const patchOperation1: models.PatchOperation = {
  op: "replace",
  path: "/title",
};

const patchOperation = [
  patchOperation1,
];

apiCaller.updateFlagLink(
  "projectKey_string", // projectKey
  "featureFlagKey_string", // featureFlagKey
  "id_string", // id
  patchOperation,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FlagLinksBetaApi#updateFlagLink:");
  console.log(error.body);
});

File: launchdarkly/python/src/UpdateFlagLinkExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    patch_operation_1 = models.PatchOperation(
        op="replace",
        path="/title",
    )

    patch_operation = [
        patch_operation_1,
    ]

    try:
        response = api.FlagLinksBetaApi(api_client).update_flag_link(
            project_key=None,
            feature_flag_key=None,
            id=None,
            patch_operation=patch_operation,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling FlagLinksBetaApi#update_flag_link: %s\n" % e)
